.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Testing Interoperability
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: workflow_dispatch
+jobs:
+  Testing_Interoperability:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Downloads assets
+        uses: robinraju/release-downloader@v1.7
+        with:
+          latest: true
+          fileName: "*"
+      - name: Unzip
+        uses: montudor/action-zip@v1
+        with:
+          args: unzip -qq *.zip -d executables
+      - name: Run Interoperability script
+        run: |
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install -r requirements.txt
+          cd executables
+          for i in ./* ; do for j in ./*; do python3 ./../interoperability_report.py -P $i -S $j -o=./../general_report.xml; done; done
+      - name: XUnit Viewer
+        id: xunit-viewer
+        uses: AutoModality/action-xunit-viewer@v1
+        with:
+          results: ./general_report.xml
+      - name: Attach the report
+        if: always()
```

Comment:
This condition does nothing
```suggestion
```

Reply:
right, i think i can delete it

Reply:
i think it is important because without it if there is an error before (with xunit-viewer an error means one of the test cases failed), the report is not going to be attached.

Reply:
I see